bin/make.sh

```diff
@@ -222,11 +222,6 @@ values() {
             PROMPT="0"
           fi
           ;;
-        TPA__*)
```

Comment:
@otaviof Why did you remove bit?

Reply:
With these lines in place all `TPA__*` was set to the single entry I was able to enter. I still need to investigate this further.